test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -21,11 +21,9 @@
  * questions.
  */
 
- /*
+/*
  * @test
  * @key headful
- * @library ../../regtesthelpers
- * @build Util
  * @bug 8033699 8154043 8167160 8208640 8226892
  * @summary  Incorrect radio button behavior when pressing tab key
  * @run main bug8033699
```

Comment:
I thought about moving this block to the start of the class. When viewing the code in IDE, you'll see the test description right away if this block is located before the class declaration. When the test description is above imports, it gets collapsed along with the copyright header.